### PR TITLE
UI and feature improvement

### PR DESCRIPTION
**Pull request by wixette**
This PR mainly includes:
* Update of 3D models and animations
* Implement the left-end and right-end levels for the out-of-bound positive numbers (try 1e-11, 1e+15, etc.)
* Add Chinese/English description texts for all the reference objects
* Add Chinese/English contents for the About dialog

This branch can be built and tested together with the `wip_wixette_scene` branch of the `SeedCalcNonOpenSource` repo.

### CODE REVIEW

File: Assets/Src/Scripts/SeedCalc/CuttingBoard.cs [hunk 1]
```diff
@@ -447,6 +493,18 @@ private class SlideAnimConfig {
...
     }
 
+    private void ScrollRainbowToLeftEnd() {
+      float texOffsetX = _rainbowTexInitOffsetX;
+      GetComponent<Renderer>().material.SetTextureOffset("_MainTex",
+                                                         new UnityEngine.Vector2(texOffsetX, 0));
+    }
+
+    private void ScrollRainbowToRightEnd() {
+      float texOffsetX = 1.0f;
```
Comment (codingpotato) [inline Assets/Src/Scripts/SeedCalc/CuttingBoard.cs:503]: Define texOffsetX as a const?

File: Assets/Src/Scripts/SeedCalc/CuttingBoard.cs [hunk 2]
```diff
@@ -447,6 +493,18 @@ private class SlideAnimConfig {
                                                          new UnityEngine.Vector2(texOffsetX, 0));
     }
 
+    private void ScrollRainbowToLeftEnd() {
+      float texOffsetX = _rainbowTexInitOffsetX;
```
Comment (codingpotato) [inline Assets/Src/Scripts/SeedCalc/CuttingBoard.cs:497]: No need texOffsetX variable? Use `_rainbowTexInitOffsetX` in following function call directly?

**Review by codingpotato — CHANGES_REQUESTED** [review]

**Comment by codingpotato** [comment]
This PR has conflicts. May need merge `main`.

File: Assets/Src/Scripts/SeedCalc/MathUtils.cs [hunk 3]
```diff
@@ -23,5 +24,17 @@ public static class MathUtils {
...
+
+    // Returns the upper bound of a positive double value's order of magnitude. For example, returns
+    // 10 for 3, 100 for 30, 0.1 for 0.03, 0.01 for 0.003. etc.
+    //
+    // A boundary number will be counted as the upper bound of its lower order of magnitude. For
+    // example, this method returns 10 for 10, 0.1 for 0.1, etc.
+    public static double OrderOfMagnitudeUpperBound(double value) {
+      Debug.Assert(value > 0);
+      double log = Math.Log10(value);
+      int power = log % 1 == 0 ? (int)log : (int)Math.Floor(log + 1);
```
[the remark this answers is not in the document]
Comment (wixette) [reply Assets/Src/Scripts/SeedCalc/MathUtils.cs:50]: No. The result of `Math.Log10(value)` is a float number. `x % 1 == 0` is to determine if x is an integer. E.g., in Python:

```
>>> import math
>>> math.log10(3) % 1
0.47712125471966244
>>> math.log10(10)%1
0.0
```

File: Assets/Src/Scripts/SeedCalc/CuttingBoard.cs (hunk 1 above)
Comment (wixette) [reply Assets/Src/Scripts/SeedCalc/CuttingBoard.cs]: Deleted the varlable.

File: Assets/Src/Scripts/SeedCalc/CuttingBoard.cs (hunk 2 above)
Comment (wixette) [reply Assets/Src/Scripts/SeedCalc/CuttingBoard.cs]: Deleted the varlable.

**Comment by wixette** [comment]
> This PR has conflicts. May need merge `main`.

Done.

**Review by codingpotato — APPROVED** [review]
LGTM

**Pull request merged**